Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -346,6 +383,7 @@ void MainWindow::StartGame(const QString& path)
   }
   ShowRenderWidget();
   emit EmulationStarted();
+  m_stop_requested = false;
```

Comment:
While putting this here should be fine, m_stop_requested should ideally be reset to false in the stopped callback, not here.